README.md

```diff
@@ -189,7 +191,7 @@ Initialize your map by copying the following code into the HTML tab of your JSFi
 
 ### Add a title and info box(front-end UI):
 
-Add the following code between the <body> opening and </body> closing tags:
+Add the following code between the `<body>` opening and `</body>` closing tags:
```

Comment:
Some of these tags were being interpreted as plain HTML and futzing up the formatting.